ElectronicObserverTypes/EquipmentId.cs

```diff
@@ -519,4 +519,5 @@ public enum EquipmentId
 	SeaplaneRecon_SeaOtter = 515,
 	Interceptor_Me262A1aR1 = 516,
 	RadarSmall_PassiveRadiolocator_E27_Type22SurfaceRadarKai4_CalibratedLateModel = 517,
+	MainGunMedium_14cmTwinGunKaiNi = 518
```

Comment:
Add a trailing comma, so this line doesn't have to get changed when the next equipment gets added.